skopt/optimizer/forest.py

```diff
@@ -7,12 +7,10 @@
 from ..learning import RandomForestRegressor
 
 
-def forest_minimize(func, dimensions, base_estimator="ET",
-                    n_calls=100, n_random_starts=10,
-                    acq_func="EI", acq_optimizer="auto",
```

Comment:
This will break user-code. Let's keep it for now and hopefully we will add something better later on?